src/BlockTypes/Cart.php

```diff
@@ -18,7 +18,7 @@ class Cart extends AbstractBlock {
 	 *
 	 * @var string
 	 */
-	protected $block_name = 'cart';
+	protected $block_name = 'cart-block';
```

Comment:
It seems a little redundant/unfortunate to have to suffix a block name with `-block` - is this necessary?